api/typescript/src/core/kurtosis_core_rpc_api_bindings/connect/api_container_service_connect.d.ts

```diff
@@ -1,4 +1,4 @@
-// @generated by protoc-gen-connect-es v1.3.0 with parameter "target=js+dts"
+// @generated by protoc-gen-connect-es v0.13.2 with parameter "target=js+dts"
```

Comment:
Your version seems quite behind.

Reply:
i have been using the nix setup - i tried changing this in package.json; evaporated flake.lock but i can't seem to fix this; will create an issue for edgy!

Reply:
closing this for now as i am tracking this in issue #2361